ext/dom/tests/DOMAttr_entity_expansion.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+DOMAttr entity expansion
+--FILE--
```

Comment:
This needs an extensions section though:

```
--EXTENSIONS--
dom
```

Reply:
It's also a good idea if you print out the value after setting it by doing `print $attr->value;`. Just for some extra robustness against regressions.

Reply:
Done.